packages/babel-parser/test/fixtures/experimental/discard-binding/array-pattern/output.json

```diff
@@ -0,0 +1,519 @@
+{
+  "type": "File",
+  "start":0,"end":566,"loc":{"start":{"line":1,"column":0,"index":0},"end":{"line":36,"column":1,"index":566}},
+  "errors": [
+    "SyntaxError: Binding invalid left-hand side in array destructuring pattern. (35:17)"
```

Comment:
Is `try {} catch ([void]) {}` intended to be invalid?

Reply:
Good catch, this is a bug in our implementation.